### ISSUE TITLE
Load Titles From Tablo Hangs on "Finishing Up"

### ISSUE THREAD

**Issue by simplicitybill** [jessedp/tablo-tools-electron#94]
**Describe the bug**
Installed Tablo 0.2.4 with Tablo Tools 0.2.4.exe. The application worked great for about 45 days, the on reload of Tablo content, the application hung on "Finishing Up". 

Installed on a second laptop to ensure I had no connectivity issues with the Tablo itself. Application loaded all Tablo content without issue. Used the 2nd laptop for about a month.  Tablo Tools threw a script error in Windows and closed while not in use. Restarted the application on the 2nd laptop, it hung as with with the first laptop on "Finishing Up". 

I uninstalled and manually verified the Program directory was deleted C:\<user>\AppData\Local\Programs\tablo-tools and reinstalled, achieved successful re-install, with a newly populated app directory.

On start > Load, got a full progress bar to 100% as before and the app hung on finishing up.

I created a new user account on the same laptop and reinstalled. Started the application and successfully loaded Tablo content and went to export. 

Main screen. After uninstalling Tablo Tools and verifying that the application directory was deleted. the application goes right back to attempting to load from the previous install.

![1st screen](https://user-images.githubusercontent.com/82282587/114281547-ed82f600-9a0c-11eb-9bd3-ea2651220537.png)

Screenshot of the Console Log

![2nd screen](https://user-images.githubusercontent.com/82282587/114281594-2de27400-9a0d-11eb-80bc-c5e4d679eae3.png)

OS: Win 10 latest build updated
Version: Tablo Tools 0.2.4 installed from exe.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment:**
 - OS: [e.g. Win 10]
 - App Version from the Home screen [e.g.0.2.1-beta.1 ]

**Additional context**

**Comment by jessedp** [comment]
Bad news: 
The error isn't something that can be fixed without a new release (it _may_ not exist under Mac/Linux, though that's likely not helpful)

Marginally better news:
I'm currently working on upgrading all the guts of the app, but it's going to take some more time to get through that. Assuming I do and get another release out, I definitely plan on working through open issues before doing so.

**Comment by simplicitybill** [comment]
Thanks Jesse. It's a great little app, I'll definitely be interested when you release the upgrade.

The workaround is pretty simple if anyone else runs into this. I was so happy to have a tool to visualize the Tablo device and perform operations on it that it made sense to me to find it.

If you keep your Tablo Export directory in the root of C:\  and if you are using anything inline like MCE Buddy before copying to the media server and keep that operation directed at your export directory from Tablo Tools, and then output from MCE Buddy to the root of C:\, all you have to do is create a new user account in Win 10, reinstall Tablo Tools to that user (simply switch users) and install on the new user desktop from the Tablo Tools exe, and just set your export directory in the new Tablo Tools install to that directory in the root of C:\.

If you've installed MCE Buddy for all users, the new user account will inherit the MCE Buddy permission and just start using Tablo Tools from the new account.

I have a sacrificial laptop that does nothing but Tablo rips and commercial removal, so this is only a minor inconvenience for me that affects nothing else and it's well worth the few minutes to setup the new user account if you run into this.

Thanks for the reply and I will definitely download the new release when it's available.

**Comment by simplicitybill** [comment]
One more thing. It happened again. It appears that if Tablo Tools is open and this time it was running on Export and Windows updates and shuts down the PC, this will trigger the issue.

**Comment by jessedp** [comment]
Doh. Try deleting ` C:\<user>\AppData\Roaming\tablo-tools`  . 

What you had seemed right and at a glance what I put on the website (which is _wrong_). After you mentioned this, though, I went and booted up Windows...

**Comment by newcxcw** [comment]
Jesse, I have the same problem, and can correct it by deleting the tablo-tools profile after backing up my .json files to save my destination settings.  I only have the hang-up on the one of my two Tablos that has a 4 TB external drive.  The other Tablo has a 1 TB drive and never hangs.

I just reloaded and it didn't hang this morning, but here are my current .db file sizes:

...\Roaming\tablo-tools

04/25/2021  10:17 AM         1,074,022 export-log.db
04/25/2021  10:10 AM                 0 saved-search.db
04/25/2021  10:09 AM            19,208 SID_5087B8009D2E-channel.db
04/25/2021  10:09 AM         3,405,330 SID_5087B8009D2E-recordings.db
04/25/2021  10:09 AM            59,618 SID_5087B8009D2E-show.db
04/25/2021  10:10 AM            13,332 SID_5087B800DB8A-channel.db
04/25/2021  10:10 AM         6,608,253 SID_5087B800DB8A-recordings.db
04/25/2021  10:10 AM           144,583 SID_5087B800DB8A-show.db
04/25/2021  10:10 AM                 0 template-naming.db

...\Roaming\tablo-tools\databases

04/21/2021  09:53 AM            28,672 Databases.db

Here is a listing of files I backed up a few days ago when it did hang:

...\Roaming\tablo-tools.backup

04/21/2021  09:53 AM                 0 export-log.db
04/21/2021  09:53 AM                 0 saved-search.db
03/08/2021  04:07 PM            60,176 SID_5087B8009D2E-channel.db
03/08/2021  04:07 PM         8,963,966 SID_5087B8009D2E-recordings.db
03/08/2021  04:07 PM           179,804 SID_5087B8009D2E-show.db
04/21/2021  09:54 AM             7,135 SID_5087B800DB8A-channel.db
04/21/2021  09:54 AM         3,835,819 SID_5087B800DB8A-recordings.db
04/21/2021  09:54 AM            77,196 SID_5087B800DB8A-show.db
04/21/2021  09:53 AM                 0 template-naming.db

...\Roaming\tablo-tools.backup\databases

02/15/2021  05:42 PM            28,672 Databases.db
